torch/_inductor/ir.py

```diff
@@ -3107,10 +3114,15 @@ def create(cls, kernel, *args, **kwargs):
         )
 
         def generate_output(output, index=""):
-            if isinstance(output, (list, tuple)):
-                return type(output)(
+            if isinstance(output, list):
+                return [
                     generate_output(output[i], f"{index}[{i}]")
                     for i in range(len(output))
+                ]
+            elif isinstance(output, tuple):
```

Comment:
is tuple encountered only with cpp wrapper? It looks strange that `list` case generates `[]` access, and `tuple` does `<>`